### DISCUSSION TITLE
Universal Style Guide

### DISCUSSION THREAD

**Discussion by tesseralis** [reactjs/reactjs.org-translation#239] [Team Posts]
I created a universal style guide here: https://github.com/reactjs/reactjs.org-translation/blob/master/style-guide.md

Let me know if you have any suggestions for it. I think I've covered the basics :)

**Comment by tesseralis** [comment]
Also @reactjs/reactjs-org-spanish-translation how's my Spanish? 😛

**Comment by carburo** [comment]
Your Spanish is perfect! 😆 
It seems pretty good to me. I am going to translate it into Spanish and add it to our repo.
